src/Nox.EntityFramework.Postgres/PostgresDatabaseProvider.cs

```diff
@@ -54,4 +56,14 @@ public string ToTableNameForSqlRaw(string table, string schema)
     {
         throw new NotImplementedException();
     }
+
```

Comment:
if this is for all providers? you can add them to the base class NoxDatabaseConfigurator

Reply:
Rather keep it on the provider, not all database provider will implement the configurator